moveit_commander/src/moveit_commander/planning_scene_interface.py

```diff
@@ -262,4 +197,95 @@ def get_attached_objects(self, object_ids = []):
             msg = AttachedCollisionObject()
             conversions.msg_from_string(msg, ser_aobjs[key])
             aobjs[key] = msg
-        return aobjs 
+        return aobjs
+
+    @staticmethod
+    def __make_existing(name):
+        """
+        Create an empty Collision Object, used when the object already exists
+        """
+        co = CollisionObject()
+        co.id = name
+        return co
+
+    @staticmethod
+    def __make_box(name, pose, size):
+        co = CollisionObject()
+        co.operation = CollisionObject.ADD
+        co.id = name
+        co.header = pose.header
+        box = SolidPrimitive()
+        box.type = SolidPrimitive.BOX
+        box.dimensions = list(size)
+        co.primitives = [box]
+        co.primitive_poses = [pose.pose]
+        return co
+
+    @staticmethod
+    def __make_mesh(name, pose, filename, scale=(1, 1, 1)):
+        co = CollisionObject()
+        if pyassimp is False:
+            raise MoveItCommanderException(
+                "Pyassimp needs patch https://launchpadlibrarian.net/319496602/patchPyassim.txt")
+        try:
+            scene = pyassimp.load(filename)
+        except pyassimp.AssimpError as e:
+            raise MoveItCommanderException(e.message)
```

Comment:
I disagree.
If you don't know how to handle an exception, don't catch it.
Instead, we should document that this can be thrown by the functions which use this.

Reply:
Fair point.

Reply:
Removed this change.